src/back/routes/notification.ts

```diff
@@ -30,6 +32,17 @@ async function sendNotification(req: WithAuth) {
     throw new RequestError('Invalid data', RequestError.BadRequest)
   }
 
+  const users = await UserModel.getDiscordIdsByAddresses([recipient])
```

Comment:
I think this should be `[isArray(recipient) ? ...recipient : recipient]` as recipient could be `string | string[]`